pkg/kubelet/eviction/helpers.go

```diff
@@ -214,6 +219,27 @@ func parseThresholdStatement(statement string) (evictionapi.Threshold, error) {
 	}, nil
 }
 
+// getAllocatableThreshold returns the thresholds applicable for the allocatable configuration
+func getAllocatableThreshold(allocatableConfig []string) []evictionapi.Threshold {
+	for _, key := range allocatableConfig {
+		if key == cm.NodeAllocatableEnforcementKey {
```

Comment:
I'd prefer exposing this logic as a containermanager method instead. why should evictions care about allocatable configuration excepting knowing if it needs to evict at allocatable level